scripts/plant_description_engine_privileges.sql

```diff
@@ -0,0 +1,13 @@
+USE `arrowhead`;
+
+REVOKE ALL, GRANT OPTION FROM 'plant_description_engine'@'localhost';
+
+GRANT ALL PRIVILEGES ON `arrowhead`.`pde_rule` TO 'plant_description_engine'@'localhost';
+GRANT ALL PRIVILEGES ON `arrowhead`.`plant_description` TO 'plant_description_engine'@'localhost';
+
```

Comment:
add privilege to log table to as your log4j config wants it.
GRANT ALL PRIVILEGES ON \`arrowhead\`.\`logs\` TO 'plant_description_engine'@'localhost';`

Reply:
Fixed.